### DISCUSSION TITLE
GPT-4 or ChatGPT+ Plus Required?

### DISCUSSION THREAD

**Discussion by mariohaya** [npiv/chatblade#82] [Q&A] [answered]
is a ChatGPT+ Plus membership required for this? I've exported my account's API key with `export OPENAI_API_KEY=` but inputting any text on chatblade prints out:

`"openai error: Error code: 429 - {'error': {'message': 'You exceeded your current quota, please check your plan and billing details. For more information on this error, read the docs: 
https://platform.openai.com/docs/guides/error-codes/api-errors.', 'type': 'insufficient_quota', 'param': None, 'code': 'insufficient_quota'}}"`

I am not using a new account and I cannot really create a new one either because I don't have a fresh cellphone number that hasn't been used in OpenAI before.

**Comment by npiv** [comment, marked as answer]
No chatgpt subscription isn't required just an api account.

But openai recently moved to a prepaid solution vs a billing one. https://community.openai.com/t/oai-api-switching-to-pre-paid-billing/659398

You likely don't have credit and need to top up.
